processor/src/main/java/org/mapstruct/ap/internal/model/common/TypeFactory.java

```diff
@@ -261,11 +261,13 @@ else if ( type.isPrimitive() ) {
      * Get the ExecutableType for given method as part of usedMapper. Possibly parameterized types in method declaration
      * will be evaluated to concrete types then.
      *
+     * <b>IMPORTANT:</b> This should only be used from the Processors. as they are operating over executable elements.
```

Comment:
Please exchange the first period for a comma... then it's ready to be sqashed and merged 👍